src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -588,6 +604,15 @@ private void handleUpdateUserFailures(User user, Handler<AsyncResult<Response>>
       return;
     }
 
+    if (isInvalidUsernameError(errorMessage)) {
+      logger.info("The user with the ID {} must have a username, since consortium mode is enabled", user.getId());
+      asyncResultHandler.handle(
+        succeededFuture(PutUsersByUserIdResponse
+          .respond400WithTextPlain(
+            String.format("The user with the ID %s must have a username, since consortium mode is enabled", user.getId()))));
```

Comment:
**Minor**
This string can be a constant since used twice and you can remove the comma before the word "since"

Reply:
Refactored